eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -38,4 +38,9 @@
     <UsagePattern IdentityGlob="System.Reflection.MetadataLoadContext/*7.0.0*" />
     <UsagePattern IdentityGlob="System.Security.Cryptography.ProtectedData/*7.0.0*" />
   </IgnorePatterns>
+  <Usages>
+    <!-- Dependencies of the legacy `rzc.dll` (see `_LegacyRazorTool` in `GenerateLayout.targets`). -->
+    <Usage Id="Microsoft.AspNetCore.Mvc.Razor.Extensions.Tooling.Internal" Version="7.0.0-preview.23330.1" IsDirectDependency="true" />
+    <Usage Id="Microsoft.CodeAnalysis.Razor.Tooling.Internal" Version="7.0.0-preview.23330.1" IsDirectDependency="true" />
```

Comment:
@dotnet/source-build-internal - we would like to have a legacy version of `rzc.dll` built here which would depend on a fixed version of Razor DLLs. Can these pre-builts be added as exceptions?